src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawerAdapter.java

```diff
@@ -23,11 +25,26 @@ public void setPosition(int x, int y) {
 
 	@Override
 	public void operateTo(int x, int y) {
-		ILine line = LineFactory.getBasicLine();
+
+		ILine line;
+		switch (drawerType){
+			case "Buggy":
+				line = LineFactory.getBasicLine();
+				break;
+			case "Dotted":
+				line = LineFactory.getDottedLine();
+				break;
+			case "Special":
+				line = LineFactory.getSpecialLine();
+				break;
```

Comment:
Unfortunately, this is not a generic solution. (e.g. new line type requires modifications in this class, unnecessary direct coupling with LineFactory)